assets/queries/terraform/tencentcloud/disk_encryption_disabled/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "1ee0f202-31da-49ba-bbce-04a989912e4b",
+  "queryName": "Disk Encryption Disabled",
+  "severity": "MEDIUM",
+  "category": "Encryption",
+  "descriptionText": "Disks should have encryption enabled",
+  "descriptionUrl": "https://registry.terraform.io/providers/tencentcloudstack/tencentcloud/latest/docs/resources/cbs_storage#encrypt",
+  "platform": "Terraform",
+  "descriptionID": "1ee0f202",
+  "cloudProvider": "tencentcloud",
+  "cwe": ""
+}
```

Comment:
Kindly add `"experimental": "true"` in this `metadata.json` file.
[Check this as an example](https://github.com/Checkmarx/kics/blob/master/assets/queries/terraform/nifcloud/computing_instance_has_common_private/metadata.json#L11).